NuGet.config

```diff
@@ -18,4 +19,124 @@
   <disabledPackageSources>
     <clear />
   </disabledPackageSources>
-</configuration>
+  <packageSourceMapping>
+    <clear />
+    <packageSource key="dotnet-eng">
+      <package pattern="microsoft.*" />
```

Comment:
I imagine this is not ideal, but won't be slower than our current implementation without packagesourcemapping. If we start to see this causing slowdowns we can be more specific